suite-native/atoms/src/Input/Input.tsx

```diff
@@ -113,7 +113,7 @@ const inputStyle = prepareNativeStyle<InputStyleProps>((utils, { isIconDisplayed
     borderWidth: 0,
     flex: 1,
     // Make the text input uniform on both platforms (https://stackoverflow.com/a/68458803/1281305)
-    paddingTop: isFocused ? utils.spacings.large : 0,
+    paddingTop: utils.spacings.large,
```

Comment:
`isFocused` is not used anymore so please remove it from style prop types and its usage

Reply:
fixed by 04ff390c6d